packages/vue/src/stories/releases/v0.12.x/v0.12.4/v0.12.4.stories.mdx

```diff
@@ -10,5 +10,6 @@ import { Meta } from "@storybook/addon-docs/blocks";
 ## 🐛 Fixes
 
 - E2E tests: SfHero on home page fixed
+- SfSidebar: add `position` props and fix animation loading for async component
```

Comment:
what do you think about adding additional information about this change? It's BC so would be good to inform about it and show what to do to avoid breaking a project :)

Reply:
> what do you think about adding additional information about this change? It's BC so would be good to inform about it and show what to do to avoid breaking a project

@justyna-13 I've added BC description in changelog. Please check it's looks ok :)

Reply:
@RafalDobrowolski great :) thanks